models/forms/ManageProposalForm.php

```diff
@@ -18,8 +18,8 @@ public function rules()
         return [
             [['title', 'content'], 'required'],
             ['relatedFile', 'file', 'extensions' =>
-                Util::UPLOADED_FILE_ALLOWED_EXTENSIONS,
-                'maxSize' => 52428800],
+                Util::UPLOADED_FILE_RULES,
```

Comment:
The extensions of allowed files are now keys of this array. So You should use `aray_keys(Util::UPLOADED_FILE_RULES)`

Reply:
Yes I am the one WHO FIND THE SOLUTION YOU BASTARD !

Reply:
It's ARRAY NOT ARAY DUDE

Reply:
I don't see what you mean